formatting/key_pair_id_test.go

```diff
@@ -0,0 +1,26 @@
+package formatting
+
+import (
+	"strconv"
+	"testing"
+)
+
+type KeypairIDTest struct {
```

Comment:
Not sure if the structure is necessary. In the other test you inlined the struct. Consistency and `fmt` alignment would be appreciated.

Reply:
Removed